nano/PKGBUILD

```diff
@@ -18,19 +18,13 @@ makedepends=('autotools'
              'libiconv-devel'
              'gettext-devel')
 backup=('etc/nanorc')
-source=(https://www.nano-editor.org/dist/v7/${pkgname}-${pkgver}.tar.xz{,.asc})
-sha256sums=('86f3442768bd2873cec693f83cdf80b4b444ad3cc14760b74361474fc87a4526'
+source=(https://www.nano-editor.org/dist/v8/${pkgname}-${pkgver}.tar.xz{,.asc})
+sha256sums=('c17f43fc0e37336b33ee50a209c701d5beb808adc2d9f089ca831b40539c9ac4'
             'SKIP')
 validpgpkeys=(
   '168E6F4297BFD7A79AFD4496514BBE2EB8E1961F' # Benno Schulenberg <bensberg@telfort.nl>
 )
 
-prepare() {
-  cd ${srcdir}/${pkgname}-${pkgver}
-
-  autoreconf -fiv
```

Comment:
any reason this got removed?

Reply:
I met an autoreconf failure. And I checked the PKGBUILD of Arch Linux, finding there is not `autoreconf`.

Reply:
ok. We autoreconf since we patch libtool, so comparing to Arch isn't useful.

But there doesn't seem to be a difference here (it builds and the filenames are the same), so good enough I guess.